lib/src/client.dart

```diff
@@ -2671,12 +2670,10 @@ class Client extends MatrixApi {
         }
         final accountData = syncRoomUpdate.accountData;
         if (accountData != null && accountData.isNotEmpty) {
-          await _handleRoomEvents(
-            room,
-            accountData,
-            EventUpdateType.accountData,
-            store: false,
-          );
+          for (final event in accountData) {
+            await database?.storeRoomAccountData(event);
```

Comment:
I think we don't need to store it in DB here because it is `LeftRoomUpdate`. Even the old code has `store` set to `false`

Reply:
Thanks, you are right again :D